compliance/portugal.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: "Invoicing compliance in Portugal"
+sidebarTitle: "🇵🇹 Portugal"
+description: Information about e-invoicing in Portugal
+---
+
+<Card title="Portugal's e-invoicing regulation timeline" size="20" icon="timeline" href="/timelines/portugal" horizontal>
+  View current and upcoming regulation  →
+</Card>
+
+## Executive summary
+
+Portugal mandates electronic invoicing for all B2B transactions starting January 1, 2026, through the **ACTUD/QR** system. This regulation requires businesses to use approved software that generates invoices with unique validation codes (ACTUD) assigned by the Portuguese Tax Authority (AT), ensuring transparency and preventing fraud. Each invoice must include a QR code that links to the AT's online verification service, allowing customers and tax authorities to validate invoice authenticity.
+
+Invocing in Portugal is based on the EN 16931 standard with CUIS extensions, requiring businesses to notify the AT of their numbering series before issuance. The AT then validates and issues unique codes for each series, creating a centralized tracking mechanism for all invoices.
+
+Portugal also requires **SAF-T (Standard Audit File for Tax)** reporting, an XML-based format used to submit accounting and invoicing data to tax authorities. This system enables standardized digital audits and tax compliance, with mandatory reporting for B2G and B2B transactions from January 1, 2024.
+
+This article provides comprehensive guidance on [ACTUD/QR invoicing requirements](#actudqr-b2b-b2g), [SAF-T reporting](#saft-pt-b2b-b2g), and [Portuguese invoicing regulations](#regulation).
+
+Invopop provides comprehensive coverage for Portuguese invoicing through dedicated apps for AT Portugal and InvoiceXpress Portugal, as well as integrations with platforms like Stripe and Chargebee. This allows businesses to issue compliant ACTUD invoices efficiently and manage white-label (B2B2B) operations.
```

Comment:
I wonder if we want to keep advertising our IX integration or we prefer to focus on our in-house one. Might be worth checking with Juan and Sam.